content/projects/metabake.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Metabake.org
+repo: metabake/_mBake
+homepage: http://www.metabake.org
+language: Pug
```

Comment:
Sorry, missed this: Pug is the template language, but the language this generator is written in should be `JavaScript`.